pkg/agent/openflow/pipeline.go

```diff
@@ -2704,7 +2708,6 @@ func (f *featureMulticast) externalMulticastReceiverFlow() binding.Flow {
 	return MulticastRoutingTable.ofTable.BuildFlow(priorityLow).
 		Cookie(f.cookieAllocator.Request(f.category).Raw()).
 		MatchProtocol(binding.ProtocolIP).
-		MatchDstIPNet(*types.McastCIDR).
```

Comment:
is this an intentional change?

Reply:
Yes, it is not needed because all the Multicast flows have already matched-DstIPNet by flow, no need to do it again.
```
table=PipelineIPClassifier, priority=210,ip,nw_dst=224.0.0.0/4 actions=resubmit(,MulticastEgressRule)
```